atd-vze/src/Components/GridExportData.js

```diff
@@ -112,24 +112,12 @@ const GridExportData = ({
     });
 
     const cleanedAndFlattenedData = flattenedData.map(item => {
-      // Some fields are presenting issues when they are exported as .csv and
-      // imported into MSFT Excel, Acces & Power BI. For now, we're just going to
-      // modify the data for export to solve issues related to returns and double-quotes
-      // In the future, we may choose to clean these values at the database level.
-
-      const columnsToClean = [
-        "rpt_street_name",
-        "rpt_street_desc",
-        "rpt_sec_street_desc",
-        "rpt_sec_street_name",
-      ];
-
-      columnsToClean.forEach(col => {
-        if (item[col]) {
-          // remove return carriage and replace with space
-          item[col] = item[col].replace(/(\r\n|\n|\r|")/gm, " ");
```

Comment:
frank and i discussed that removing all newlines and carriage returns for these columns "rpt_street_name", "rpt_street_desc", "rpt_sec_street_desc", and "rpt_sec_street_name" is altering the data somewhat and maybe it would be best to not do that. newlines are allowed in CSV format guidelines and they dont cause apps like excel to freak out like the double quotes do (the rec_updates column often has newlines and they format fine in excel). this can be opened up for discussion though if anybody has other thoughts!

Reply:
@roseeichelmann I thought that I dropped the link to the original PR https://github.com/cityofaustin/atd-vz-data/pull/649 for these changes in response to [this comment](https://github.com/cityofaustin/atd-vz-data/pull/1248#issuecomment-1631474955) the other day, but I guess that I forgot!  The only thing that concerns me about taking this out is that it seems like this solved an issue that Xavier was facing. 

There is no GH issue attached to that PR so not super helpful, but maybe there is risk of regression for the VZ team? There are a couple example crash record IDs listed in that PR.

Reply:
ooh yeah thats a good point! i tested exporting a csv containing that crash record (17272765) with the code changes i made in this pr and none of the rpt fields for it seem to have newlines? i wonder if maybe the data has changed.

Reply:
If we have new lines in data that should not have new lines in it, like for example, a street name, then I think we should address that data in the DB and get it corrected. LDM will make this super easy too, in the future.

There are fields where newlines are valid and useful data, like the notes fields that are now being generated by the VZ team. I am going to make a note to look at this first thing in the morning and see if i can learn more about what was causing the VZ team headaches on it, and I will also possible reach out to Xavier for more background.

I guess put another way, I am keen to not drop new lines, but, of course, I obviously don't want regressions either.

It's a little late in the day to jump into an investigation, but I would be very eager to explore some solutions that let us not strip out newlines, but also ensure that there are none where they do not belong. Don't let me hold this up, of course, and I will see if i can offer something more helpful than this opinion tomorrow.

Thank you Rose & Mike!!

Reply:
I was curious about this code block so I looked into these columns.

- `rpt_street_name`
    - newline
        - none
    - double quotes
        - none
    - carriage return
        - none

- `rpt_street_desc`
    - newline
        - none
    - double quotes
        - 11243056, 11415653, 11459597, 11895097, 11988003, 14648733, 16565337, 16795451, 17069323, 17569955, 18385294, 18582777, 18627255, 19263271
    - carriage return
         - none

- `rpt_sec_street_desc`
    - newline
        - none
    - double quotes
        - 13308293, 15437978, 16202128, 16388510, 16452903, 16504241, 16529694, 16795451, 17471777, 18385294
    - carriage return
        - none

- `rpt_sec_street_name`
    - newline
        - none
    - double quotes
        - none
    - carriage return
        - none

- `All crash table columns`
    - newline
        - 18004825 has `...lso \northbound on manor...` in `investigator_narrative_ocr` column (probably a typo)
    - double quotes
        - couldn’t narrow down this search enough but we have examples above
    - carriage return
        - none